src/osd/OSDMap.cc

```diff
@@ -3112,7 +3113,8 @@ class OSDTreePlainDumper : public CrushTreeDumper::Dumper<TextTable> {
       } else {
 	*tbl << (osdmap->is_up(qi.id) ? "up" : "down")
```

Comment:
Do we allow a combination of "**down + destroyed**" here? Or just "**destroyed**" will be  good enough since it implicitly indicates that it is already done?